README.md

```diff
@@ -1,19 +1,137 @@
-![Build](https://github.com/apoelstra/rust-simplicity/workflows/Continuous%20integration/badge.svg)
+<p>
+  <a href="https://crates.io/crates/simplicity-lang"><img alt="Crate Info" src="https://img.shields.io/crates/v/simplicity-lang.svg"/></a>
+  <a href="https://github.com/BlockstreamResearch/rust-simplicity/blob/master/LICENSE"><img alt="CC0 1.0 Universal Licensed" src="https://img.shields.io/badge/license-CC0--1.0-blue.svg"/></a>
+  <a href="https://github.com/BlockstreamResearch/rust-simplicity/actions?query=workflow%3AContinuous%20integration"><img alt="CI Status" src="https://img.shields.io/github/actions/workflow/status/BlockstreamResearch/rust-simplicity/main.yml"></a>
+  <a href="https://docs.rs/simplicity-lang"><img alt="API Docs" src="https://img.shields.io/badge/docs.rs-simplicity_lang-green"/></a>
+</p>
 
 # rust-simplicity
-Under development....
 
-# Minimum Supported Rust Version
+This is the official Rust library of the [Simplicity Language](https://simplicity-lang.org/).
 
-The MSRV of this crate is **1.78.0**.
+Simplicity is a low-level, typeed functional language designed to be a drop-in replacement
```

Comment:
Not `replacement` but more like `alternative`.

Reply:
Sure, I think "drop-in alternative" reads ok. I agree that "replacement" is too agressive a term.